api/src/main/resources/application.yml

```diff
@@ -3,18 +3,19 @@ server:
 
 spring:
   datasource:
-    url: jdbc:postgresql://streamer-contracts-db.c3cposncqfx6.us-east-1.rds.amazonaws.com:5432/StreamerContractsDB
-    username: StreamerContractsAdmin
-    password: ChangeMe
-    driver-class-name: org.postgresql.Driver
-    validation-query: SELECT 1
-    initSQL: SET NAMES 'utf8mb4'
```

Comment:
why was validation-query and initSQL removed?

Reply:
Removed them when I was going crazy as to why the code wasn't working. I can add them back in.